README.md

```diff
@@ -1,30 +1,45 @@
+share and adapt the material, in particular to use it for both commercial and non-commercial security purposes, under
+the terms of: attribution to Infoblox and the license. For more details, see the LICENSE file in our repo or visit
 https://creativecommons.org/licenses/by/4.0/
 
 ## ukraine
-This folder contains IOCs related to the Russian invasion of Ukraine. 
-The majority of the content is based on Infoblox internal analytics and validation analysis, though some OSINT is also 
-included.  Our references should clearly indicate indicators that originated in OSINT. 
+This folder contains IOCs related to the Russian invasion of Ukraine.
+The majority of the content is based on Infoblox internal analytics and validation analysis, though some OSINT is also
+included.  Our references should clearly indicate indicators that originated in OSINT.
 
-The file `ukraine_russia_malicous_suspicious_iocs.csv` contains malicious and suspicious IOCs that can cause harm to 
+The file `ukraine_russia_malicous_suspicious_iocs.csv` contains malicious and suspicious IOCs that can cause harm to
 businesses and innocent users. Infoblox recommends blocking traffic from network indicators described in this file.
 
-The `ukraine_russia_legitimate_iocs.csv` file contains confirmed indicators that, 
-at the time of review, were not evidently associated with malicious activity. The related websites did not show indications 
-of hosted malware or fraudulent behavior, but may host content that is not wanted by some users. 
-This includes domains belonging to well-known donation sites providing support to Ukrainian civilians, or newly 
+The `ukraine_russia_legitimate_iocs.csv` file contains confirmed indicators that,
+at the time of review, were not evidently associated with malicious activity. The related websites did not show indications
+of hosted malware or fraudulent behavior, but may host content that is not wanted by some users.
+This includes domains belonging to well-known donation sites providing support to Ukrainian civilians, or newly
 created support programs operated by entities positively acknowledged by the online community.  Many of these are
-blocked by other vendors due to their new registration or other automated analytics. 
+blocked by other vendors due to their new registration or other automated analytics.
+
+## turkiye
+This folder contains IOCs related to the earthquake that struck Türkiye and parts of Syria in early February, 2023.
+The majority of the content is based on Infoblox internal analytics and validation analysis, though some OSINT is also
+included.  Our references should clearly indicate indicators that originated in OSINT.
+
+The file `turkiye_earthquake_legitimate_iocs.csv` contains malicious and suspicious IOCs that can cause harm to
+businesses and innocent users. Infoblox recommends blocking traffic from network indicators described in this file.
+
```

Comment:
should the description be: ...contains confirmed indicators that, a the time of review, were not evidently associated with malicious or suspicious activity. The related domains did not show indications of hosted malware or fraudulent behavior.

Reply:
file should just have the legitimate indicators, right?

Reply:
oops, good catch!